src/rp2_common/hardware_dma/include/hardware/dma.h

```diff
@@ -34,8 +34,12 @@ extern "C" {
  * * Memory to memory
  */
 
-// this is not defined in generated dreq.h
-#define DREQ_FORCE  63
+// these are not defined in generated dreq.h
+#define DREQ_DMA_TIMER0 0x3b
+#define DREQ_DMA_TIMER1 0x3c
+#define DREQ_DMA_TIMER2 0x3d
+#define DREQ_DMA_TIMER3 0x3e
+#define DREQ_FORCE      0x3f
```

Comment:
I did a bit of rummaging, and it looks like these values _are_ actually avalaible at https://github.com/raspberrypi/pico-sdk/blob/develop/src/rp2040/hardware_regs/include/hardware/regs/dma.h#L176 ? :shrug: 
Although `DREQ_DMA_TIMER1` is obviously a lot more user-friendly than `DMA_CH0_CTRL_TRIG_TREQ_SEL_VALUE_TIMER1` :laughing:

Reply:
ah nice; maybe i'll define them based on the other then